src/tests/Regressions/coreclr/GitHub_42732/test42732.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <CLRTestPriority>0</CLRTestPriority>
```

Comment:
Change to Pri1 before checkin, but leave as Pri0 for initial test pass to make sure that this works on all architectures and OSs.